src/test/java/cart/controller/ProductControllerTest.java

```diff
@@ -0,0 +1,76 @@
+package cart.controller;
+
+import cart.dto.ProductRequest;
+import cart.service.ProductService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.List;
+
+import static cart.fixture.ProductFixture.CHICKEN_RESPONSE;
+import static cart.fixture.ProductFixture.SNACK_RESPONSE;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(ProductController.class)
+class ProductControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private ProductService productService;
+
+    @Test
+    void 저장_요청() throws Exception {
+        ProductRequest request = new ProductRequest("img", "name", 1000);
+        String jsonRequest = objectMapper.writeValueAsString(request);
+
+        when(productService.findAll())
+                .thenReturn(List.of(SNACK_RESPONSE));
+
+        mockMvc.perform(post("/products")
+                        .content(jsonRequest)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(status().isCreated());
```

Comment:
```suggestion
                        .contentType(MediaType.APPLICATION_JSON_VALUE))
                 .andDo(print())
                .andExpect(status().isCreated());
```
프린트도 찍어 실제 요청된 값과 응답값을 확인하시는 걸 권장합니다 👍 테스트는 기능명세의 의미도 있어서 결과가 이해하기 쉬운 형태일수록 좋습니다